src/main/java/io/debezium/connector/vitess/VitessValueConverter.java

```diff
@@ -34,9 +37,13 @@ public VitessValueConverter(
     @Override
     public SchemaBuilder schemaBuilder(Column column) {
         String typeName = column.typeName().toUpperCase();
-        if (matches(typeName, "JSON")) {
+        if (matches(typeName, Query.Type.JSON.name())) {
             return Json.builder();
         }
+        if (matches(typeName, Query.Type.ENUM.name())) {
+            String commaSeperatedOptions = Strings.join(",", column.enumValues());
+            return io.debezium.data.Enum.builder(commaSeperatedOptions);
```

Comment:
How about we add an overload to `build()` which takes the values, making usage for callers like this a bit simpler?

Reply:
Of course. I'll create a PR to overload the `builder()` static function in the Debezium main repo.

Reply:
I added the `builder()` overload to the existing Debezium main repo [PR](https://github.com/debezium/debezium/pull/2142). Would that be ok? I could also create a separate ticket and PR.

Reply:
I'll wait for the Debezium PR to be merged and built use the overloaded `builder()` here.

Reply:
Just merged the core PR :)